docker/serverDockerfile

```diff
@@ -20,4 +20,5 @@ RUN chmod +x /usr/local/bin/docker-entrypoint.sh
 ENTRYPOINT ["docker-entrypoint.sh"]
 
 EXPOSE 3306 33060
+WORKDIR /var/lib/dolt
```

Comment:
dolt sql and dolt sql-server are need to use the same directory, so I set it here